syntax = "proto3";

package recommender;

service Par2Vec {
    rpc convertParToVec(Par2VecRequest) returns (DefaultReply) {}
}

message Par2VecRequest {
    string paragraph  = 1;
}

message DefaultReply {
    uint32  dim         = 1;
    repeated float fVec = 2;
}

